go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/clarketm/json v1.17.1
 	github.com/coreos/go-semver v0.3.1
 	github.com/coreos/go-systemd/v22 v22.5.0
-	github.com/coreos/ignition/v2 v2.14.1-0.20230218211505-da6c29bdd24f
+	github.com/coreos/ignition/v2 v2.15.1-0.20230306202950-5c31ef8eb367
```

Comment:
Hmm, why revendor to Git `main` rather than the latest release?  If we pin to a Git commit, Dependabot won't automatically update to new releases in the future.